encode_utils/MetaDataRegistration/eu_register.py

```diff
@@ -162,11 +162,7 @@ def main():
     no_aliases = args.no_aliases
     overwrite_array_values = args.overwrite_array_values
 
-    if dcc_mode:
-        conn = euc.Connection(dcc_mode, dry_run)
-    else:
-        # Default dcc_mode taken from environment variable DCC_MODE.
-        conn = euc.Connection()
+    conn = euc.Connection(dcc_mode, dry_run)
```

Comment:
I think there is no need for this if branch as `dcc_mode` is `None` if not passed via cli

Reply:
Yeah looks like it.